pkgs/racket-test-core/tests/racket/core-tests.rktl

```diff
@@ -25,7 +25,11 @@
 (load-relative "deep.rktl")
 (load-relative "contmark.rktl")
 (load-relative "prompt.rktl")
-(load-relative "will.rktl")
+(unless (eq? (system-type 'gc) 'cgc)
```

Comment:
Should we just do this for the specific test that fails?
Also, we could just mark this whole test as allowed-to-fail, right?

Reply:
We could do that just for the test that fails, if it's simple to find out which test that is. I have honestly not looked at the code for `will.rktl`.

Regarding marking the test as allowed-to-fail, I am not sure in GitHub yet but when you do that in GitLab, the whole CI is marked as yellow and I found that really makes the whole system pointless because if a job is marked as yellow, the whole pipeline is marked as yellow it makes it much harder to understand if the root cause for the CI yellowness changed, or not. However, this is speaking from GitLab experience, not GitHub. Unless you are really against this, I would prefer to mark for now the test as skipped on `cgc`.

Reply:
Just pushed a change to move the skip check into `will.rktl`. Unfortunately finding the `sync` tests to skip is proving to be harder. Is there any good way to know in which line the failing tests are in?

Reply:
Basically you look for calls that seem similar to the failures. Otherwise you run the file directly and look at where the failure appears in the large amount of output.

Reply:
Ah - thanks. I think wtih `sync` with need to go the generate large amount of output and explore. :-)